tensorvision/train.py

```diff
@@ -135,20 +135,18 @@ def maybe_download_and_extract(hypes):
 def _write_precision_to_summary(precision, summary_writer, name, global_step,
                                 sess):
     """
-    TODO.
+    Write the precision to the summary file.
 
     Parameters
     ----------
-    precision : TODO
-        TODO
-    summary_writer : TODO
-        TODO
-    name : TODO
-        TODO
-    global_step : TODO
-        TODO
-    sess : TODO
-        TODO
+    precision : tensor
+        precision
```

Comment:
Line 143 is not adding any value. Please remove it or explain there what precision is. 
(If you want / need math mode forthat, you can do so like this: https://github.com/Lasagne/Lasagne/blob/master/lasagne/objectives.py#L97 )